application/controllers/admin/export.php

```diff
@@ -963,6 +969,13 @@ private function _exportarchive($iSurveyID, $bSendToBrowser = true)
     {
         $survey = Survey::model()->findByPk($iSurveyID);
 
+        if (
+            ($survey->hasTokensTable && !Permission::model()->hasSurveyPermission($iSurveyID, 'tokens', 'export'))
```

Comment:
Private function, but if you add Permission control here, you must control surveycontent/export too :)

Reply:
OK done their : https://github.com/LimeSurvey/LimeSurvey/blob/24d240d3a8a94086b4ff8fe00c32e507c72dcaf0/application/controllers/admin/export.php#L41

this file need a rewrite …